ctapipe/image/pixel_likelihood.py

```diff
@@ -0,0 +1,286 @@
+# Class for calculation of likelihood of a pixel expectation, given the pixel amplitude
+# and typically the level of noise in the pixel
+import numpy as np
+import math
+from scipy.misc import factorial
+from scipy.integrate import quad
+
+
```

Comment:
need to add an `__all__` array here to say which functions you want to export to the `ctapipe.image` namespace (right now you export everything, including the scipy and math libraries, which causes some API documentation issues)